src/components/Display.js

```diff
@@ -1,14 +1,30 @@
 import PropTypes from 'prop-types';
 import '../stylesheet/Display.css';
 
-const Display = ({ result }) => <div className="Display">{result}</div>;
-
-Display.defaultProps = {
-  result: '0',
+const Display = ({ result }) => {
+  const { total, next, operation } = result;
+  let totalDisplay = total;
+  let nextDisplay = next;
+  if (totalDisplay !== null && String(totalDisplay).indexOf('-') !== -1) {
+    totalDisplay = `(${total})`;
+  }
+  if (nextDisplay !== null && String(nextDisplay).indexOf('-') !== -1) {
+    nextDisplay = `(${next})`;
+  }
+  const display = `${totalDisplay == null ? '' : totalDisplay}
+    ${operation == null ? '' : operation}
+    ${nextDisplay == null ? '' : nextDisplay}`;
+  return (
+    <div className="Display">{display == null ? 0 : display}</div>
+  );
 };
 
 Display.propTypes = {
-  result: PropTypes.string,
+  result: PropTypes.shape({
+    total: PropTypes.string,
+    next: PropTypes.string,
+    operation: PropTypes.string,
+  }).isRequired,
```

Comment:
## Fix this prop-error validation
- [ ] Hello, we have some informing errors in the console. Kindly fix these prop validations.
![prop-error](https://user-images.githubusercontent.com/22345961/107817477-40328f80-6d87-11eb-9727-3b5a138d6771.png)